src/Tribe/RSVP.php

```diff
@@ -1539,6 +1539,19 @@ public function get_tickets_ids( $event_id ) {
 	}
 
 	/**
+	* Renders the advanced fields in the new/edit ticket form.
+	* Using the method, providers can add as many fields as
+	* they want, specific to their implementation.
+	 *
+	 * @deprecated TBD
+	 *
+	 * @return void
+	 */
+	public function do_metabox_advanced_options() {
+		_deprecated_function( __METHOD__, 'TBD', 'Tribe__Tickets__RSVP::do_metabox_advanced_options' );
```

Comment:
I think the replacement should be `'Tribe__Tickets__RSVP::do_metabox_CAPACITY_options'`? Without the funky capitalization I added for effect?